tests/client/test_version.py

```diff
@@ -0,0 +1,13 @@
+# pylint: disable=invalid-name
+
+import re
+
+from tests import BASE_URL, MASTER_KEY
+from meilisearch.version import *
+
+
+def test_get_version():
+    assert re.match(r'^(\d+\.)?(\d+\.)?(\*|\d+)$', VERSION)
+
+def test_get_qualified_version():
```

Comment:
This function doesn't seem necessary so if it is removed this test can also be removed.